app/forms/firewall-rules-create.tsx

```diff
@@ -313,11 +317,12 @@ export const CommonFields = ({ error, control }: CommonFieldsProps) => {
           <Button
             size="sm"
             onClick={hostForm.handleSubmit(({ type, value }) => {
+              // ignore click if it's a duplicate
               // TODO: show error instead of ignoring click
               if (
                 type &&
                 value &&
-                !hosts.value.some((t) => t.value === value || t.type === type)
+                !hosts.value.some((t) => t.value === value && t.type === type)
```

Comment:
this is a very similar bug that has been there since Oct 2022, preventing anyone from adding two host filters with the same value or the same type (or both, which is what we actually want)

Reply:
I learned from the best